### DISCUSSION TITLE
Problem with Cowrie Top URI Downloads Dashboard

### DISCUSSION THREAD

**Discussion by ptester87** [telekom-security/tpotce#1221] [General]
Having issues setting Cowrie dashboard. It says these fields no longer exists. Is there a way to fix this?
![image](https://user-images.githubusercontent.com/45849353/206647396-ac68ac82-2de9-4c5e-a1be-90d9d69cc494.png)

**Comment by yahavsw** [comment]
I'm having the same issue

**Reply by t3chn0m4g3** [reply]
This is to be expected as long no hits have populated the field in the index.

**Reply by yahavsw** [reply]
You are right.. it's a bit weird of an error but it does work if there is a value inside.

**Comment by pedrobart** [comment]
same issue. help
